content/blobstore-ca-rotation.md

```diff
@@ -0,0 +1,224 @@
+    certificate: ((blobstore_server_tls_2.certificate))
+    private_key: ((blobstore_server_tls_2.private_key))
+
+- type: replace
+  path: /variables/-
+  value:
+    name: blobstore_ca_2
+    type: certificate
+    options:
+      is_ca: true
+      common_name: default.blobstore-ca.bosh-internal
+
+- type: replace
+  path: /variables/-
+  value:
+    name: blobstore_server_tls_2
+    type: certificate
+    options:
+      ca: blobstore_ca_2
+      common_name: ((internal_ip))
+      alternative_names: [((internal_ip))]
+```
+
+### Step 4: Recreate all VMs, for each deployment. {: #step-4}
+
+Recreating all the VMs will remove the old CA from them. The usual way to do this is:
+
+```shell
+$ bosh -d deployment-name recreate
+```
+
+Other BOSH commands can be used to recreate the VMs, while others will restart the VMs without recreating them. Please take note of the remarks below. 
+
+
+#### Commands that will reset the blobstore configuration on the deployed VMs
+  - stop hard and start VMs
+```
+$ bosh -d deployment-name stop --hard
+$ bosh -d deployment-name start
+```
+  - recreate VMs
+```
+$ bosh -d deployment-name recreate
+```
+
+#### Commands that will NOT reset the blobstore configuration on the deployed VM
+  - restart VMs
+```
+$ bosh -d deployment-name restart
+```
+  - just stop and start VMs
+```
+$ bosh -d deployment-name stop
+$ bosh -d deployment-name start
+```
+
+
+## Troubleshooting
+
+* Performing a redeploy across one or more instances that continue to use the old CA will cause the deployment to fail using the smaller deployment technique.
+
+```
+Task 135 | 14:43:43 | Updating instance zookeeper: zookeeper/c7f03a6d-fcde-4d85-874f-8cb1503082f6 (0) (canary) (00:00:01)
+                    L Error: Action Failed get_task:
+                    Task 968fad85-0f1a-494b-6040-5cc949555d17 result:
+                    Preparing apply spec: Preparing package openjdk-8:
+                    Fetching package blob: Getting blob from inner blobstore:
+                    Getting blob from inner blobstore: Shelling out to bosh-blobstore-dav cli:
+                    Running command: 'bosh-blobstore-dav -c /var/vcap/bosh/etc/blobstore-dav.json get d1bccd47-95ad-4516-49bc-0cf42a2782c3 /var/vcap/data/tmp/bosh-blobstore-externalBlobstore-Get731225442',
+                    stdout: 'Error running app - Getting dav blob d1bccd47-95ad-4516-49bc-0cf42a2782c3:
+                    Get https://10.0.1.6:25250/d1/d1bccd47-95ad-4516-49bc-0cf42a2782c3:
+                    x509: certificate signed by unknown authority (possibly because of
+                    "crypto/rsa: verification error" while trying to verify
+                    candidate authority certificate "default.blobstore-ca.bosh-internal")', stderr: '': exit status 1
+```
+
+Any instances that have not been recreated with `bosh recreate` or a redeploy which causes a recreate will cause the above error. Perform a `bosh recreate` on any instances impacting a redeploy.
```

Comment:
add --fix here, as bosh recreate without fix will attempt to drain which won't be possible